add a function to truncate text after eos token

nairbv opened the pull request.

raghukiran1224 commented on fms/utils/generation.py:
```diff
@@ -85,3 +85,15 @@ def generate(
         result = result[0]
     return result
+
+
```
A brief description of this function will be nice to have.

raghukiran1224 replied:
we can add this later..

nairbv commented:
> left a minor comment, otherwise.. lgtm..

added the suggested comment, and fixed the lint

> one question: How specific is this to LlaM 2?

not specific at all, just a helper function for inference and validation.

(waiting for PR to update, there seems to be a delay right now).

raghukiran1224 reviewed (approved):
lgtm
